Mathlib/CategoryTheory/Monad/Limits.lean

```diff
@@ -406,4 +408,330 @@ noncomputable def leftAdjointPreservesTerminalOfReflective (R : D ⥤ C) [Reflec
 
 end
 
+-- We dualise all of the above for comonads.
+namespace Comonad
+
+variable {T : Comonad C}
+
+namespace ForgetCreatesColimits'
+
+variable (D : J ⥤ Coalgebra T) (c : Cocone (D ⋙ T.forget)) (t : IsColimit c)
+
+/-- (Impl) The natural transformation used to define the new cocone -/
+@[simps]
+def γ : D ⋙ T.forget ⟶ D ⋙ T.forget ⋙ ↑T  where app j := (D.obj j).a
+
+/-- (Impl) This new cocone is used to construct the coalgebra structure -/
+@[simps! ι_app]
+def newCocone : Cocone (D ⋙ forget T) where
+  pt := T.obj c.pt
+  ι := γ D ≫ whiskerRight c.ι (T : C ⥤ C) ≫ (Functor.constComp J _ (T : C ⥤ C)).hom
+
+/-- The coalgebra structure which will be the apex of the new colimit cone for `D`. -/
```

Comment:
Isn't it called something other than "apex" for cocones? Alternatively, you can use the mathlib terminology "cocone point".

Reply:
I think "nadir" is an option, but since we use "apex" and not "zenith" for cones, I prefer "(co)cone point." I'll change both.